src/Exception/GraphResponseException.php

```diff
@@ -102,8 +133,8 @@ public function getRawResponseBody(): array {
      * @return ODataErrorContent|null
      */
     public function getError(): ?ODataErrorContent {
-        if (array_key_exists("error", $this->responseBody)) {
-            return new ODataErrorContent($this->responseBody["error"]);
+        if (array_key_exists("error", $this->jsonBody)) {
```

Comment:
> getError() depends on $this->jsonBody to be set. From what I've seen, it only gets set with getResponseBodyJson. Do we need to first try to populate $this->jsonBody before we attempt to check for the `error` object?

Decided to set jsonBody in the constructor since we hopefully expect that Graph's response payloads for errors are not too large